code/modules/mob/mob.dm

```diff
@@ -1173,15 +1173,20 @@ GLOBAL_LIST_INIT(slot_equipment_priority, list( \
 	if(stat==DEAD)
 		return
 	var/turf/location = loc
```

Comment:
this should probably be moved inside the if statement as well. Other than that it looks good.

Reply:
moved it